### PR TITLE
Link to wiki for changeset comments

### PR DESCRIPTION
**Pull request by kepta**
Added the link:http://wiki.openstreetmap.org/wiki/Good_changeset_comments
below the comment box

Issue: https://github.com/openstreetmap/iD/issues/2923

### CODE REVIEW

**Comment by kepta** [comment]
@bhousel I wonder if the approach I took to link to localised version of OSM wiki is correct.

File: data/core.yaml [hunk 1]
```diff
@@ -203,6 +203,8 @@ en:
     modified: Modified
     deleted: Deleted
     created: Created
+    about_changeset_comments: About chageset comments
```
Comment (bhousel) [inline data/core.yaml]: "changeset"

**Comment by bhousel** [comment]
Looks good, can you fix the misspelling then I'll merge it..

**Comment by kepta** [comment]
@bhousel fixed the spelling.

**Comment by kepta** [comment]
@bhousel review please.

**Comment by bhousel** [comment]
I don't see the "Details" link when I test with the latest change..

**Comment by bhousel** [comment]
Sorry I was wrong, it was there - nice work!
merged as 427d08a, 459f292

**Pull request closed without merging**